app-backend/common/src/main/scala/S3.scala

```diff
@@ -0,0 +1,37 @@
+package com.azavea.rf.common
+
+import scala.util.{Success, Failure, Try}
+
+import org.apache.commons.io.IOUtils
+
+import com.amazonaws.auth._
+import com.amazonaws.services.s3.{AmazonS3Client, AmazonS3URI}
+import com.amazonaws.services.s3.model.{S3Object, ObjectMetadata}
+
+import java.io._
+import java.net._
+
+package object S3 {
```

Comment:
How I'm looking at it is:

The benefit of profusion of mini-projects is that if we get the groupings right in advance we get organization for free when the project is big and refactors touch more things. The cost is setting up another project folder and if we get groupings wrong we have a bigger space of projects to argue about reorganizing things into.

The benefit of the `common` project is we have a resting place for things that don't really have an intuitive home now at the cost of having to find a time to pull the trigger to reorganize later when it has grown too much. I think another cost is that a lot of things sound like they belong in `common` once `common` exists, and bookkeeping on "things that are currently in `common` that might belong together somewhere else" is tedious.

I think figuring out when to pull the trigger probably isn't easy, and if we delay too long that reorganization could be pretty costly. Obviously the compiler will help a lot so maybe the tail risk isn't actually that bad.